### DISCUSSION TITLE
PathParams class name is confusing

### DISCUSSION THREAD

**Discussion by drnextgis** [developmentseed/titiler#257] [General]
FastAPI names`Path Parameters` [parameters](https://fastapi.tiangolo.com/tutorial/path-params/) defined in URL. TiTiler has class [PathParams](https://github.com/developmentseed/titiler/blob/54d2b203860df35aff7fe9b01beaa2e35939d0e9/titiler/dependencies.py#L79) but "Path" in this case is related to the dataset path and I find this a little bit confusing. What do you think about renaming it to e.g. `DatasetPathParams`?

**Comment by vincentsarago** [comment]
That's a good idea
I think we can also change this dependency to a callable (as the TMSParams, because it's only used for the url

**Comment by vincentsarago** [comment]
This was fixed in 0.2.0
